docs/usage.rst

```diff
@@ -411,6 +411,39 @@ Additional Notes
 
 Here are misc. notes about topics that are maybe not covered in enough detail in the usage section.
 
+Item flags
```

Comment:
Yes, it is for create, but I thought it is better in the separate section to not make the create section too long.